ci-operator/config/openshift/release/openshift-release-master__nightly-4.15.yaml

```diff
@@ -96,10 +96,9 @@ releases:
       stream: nightly
       version: "4.15"
   oldest-supported-z-rollback:
-    candidate:
-      product: ocp
-      stream: nightly
-      version: "4.15"
+    release:
+      channel: candidate
+      version: 4.15.0-rc.0
```

Comment:
Using RC0 makes sense now, do we have an idea about how do we intend to bump this? Do we keep this on RC0 until GA and then we make this GA, or will we keep bumping it to RCs?

Reply:
We keep it on RC0 until GA and then we bump it to the first GA build.

Reply:
Sounds good!